docs/index.md

```diff
@@ -37,6 +37,14 @@ If you are struggling with Docker, [Docker Troubleshooting](./docker/docker-trou
 * [Contributing](./contributing/index.md)
     * Guidelines and handy information for Shopsys Framework contributors.
 
+## Frontend
+* [Design implementation and Customization](./frontend/design-implementation-and-customization.md)
+* [Introduction to LESS](./frontend/introduction-to-less.md)
+* [Configuring Image Sizes for Individual Devices Width](./frontend/configuring-image-sizes-for-individual-devices-width.md)
+* [Frontend Troubleshooting](./frontend/frontend-troubleshooting.md)
+* [Understanding the Style Directory](./frontend/understanding-the-style-directory.md)
+* [Basic design styleguide](./frontend/styleguide.md)
```

Comment:
This should be named same way as is title in the referenced file

Reply:
renamed according file